website_livechat_firstname/views/website_livechat_firstname_view.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright <YEAR(S)> <AUTHOR(S)>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <data>
+        <template id="website_livechat.channel_page" name="Livechat Channel Satisfaction Page" page="True">
```

Comment:
Instead of reimplementing, we should inherit and change only what we need. Otherwise, we are conflicting with other modules that could want to make a change to this page

```
<template id="channel_page" inherit_id="website_livechat.channel_page" >
```

Then you use xpath to target the element you want, and change some things - http://www.odoo.yenthevg.com/xpath-expressions-in-odoo-8/

Pseudo:
```
<xpath expr="//t[@t-esc='user.partner_id.name']" position="attributes">
    <attribute name="t-esc">user.partner_id.firstname</attribute>
</xpath>
```